tools/jar2shrinkwrap/src/main/java/jakartatck/jar2shrinkwrap/ClassNameRemapping.java

```diff
@@ -0,0 +1,20 @@
+package jakartatck.jar2shrinkwrap;
+
+/**
+ * Class name remapping utility that automatically can change EE 10 => EE 11 TCK test package names and/or class names.
+ *
+ * @author Scott Marlow
+ */
+public interface ClassNameRemapping {
```

Comment:
I added this from work with the jar2shrinkwrap library but can probably drop it.